inst/rmd/qc_report.rmd

```diff
@@ -0,0 +1,32 @@
+---
+params:
+  sample: Example
+  sce: !r SingleCellExperiment::SingleCellExperiment()
+  date: !r Sys.Date()
```

Comment:
So `!r` means these are R expressions I take it.

Reply:
Yep. Seems to be required for params, which is different from the other YAML headers. Maybe. I actually don't remember if I tested all combinations. One thing that is true is that `::` notation is required for params, as they are executed before any libraries are loaded.